content/blog/announcing-kustomize-support/index.md

```diff
@@ -0,0 +1,198 @@
+---
+title: "Announcing Kustomize Support"
+date: 2020-07-06
+draft: true
+meta_desc: "Pulumi adds Kustomize support to their Kubernetes provider"
+meta_image: meta.png
+authors:
+    - levi-blackstone
+
+tags:
+    - kubernetes
+---
+
+Kubernetes is complex, and there are many ways to manage Kubernetes resources. Pulumi supports many of these options, including native code SDKs, YAML, Helm, and now, Kustomize.
+
+<!--more-->
+
+Over the past year, we've heard from many users that they'd like to manage Kustomize resources directly with Pulumi.
+The [v2.4.0 release] of our Kubernetes provider adds support for Kustomize to all of our SDKs (Go, TypeScript, .NET, Python).
+
+
+## Working with local files
+
+Let's see how to deploy a Kustomization from a local directory.
```

Comment:
Question: Does the kustomize directory have to be relative to the project directory or can you give a full path? Also what does the directory structure look like with overlay files?

Reply:
> Does the kustomize directory have to be relative to the project directory or can you give a full path?

Any path will work, both relative and absolute.

> Also what does the directory structure look like with overlay files?

That's specific to kustomize; we just need the top-level kustomize directory, and the tool handles the internal layout of the directory. 

Here's an example from the upstream repo:
```
~/someApp
├── base
│   ├── deployment.yaml
│   ├── kustomization.yaml
│   └── service.yaml
└── overlays
    ├── development
    │   ├── cpu_count.yaml
    │   ├── kustomization.yaml
    │   └── replica_count.yaml
    └── production
        ├── cpu_count.yaml
        ├── kustomization.yaml
        └── replica_count.yaml
```